### DISCUSSION TITLE
Customize frontmatter field names

### DISCUSSION THREAD

**Discussion by AtlcatlQ** [davish/obsidian-full-calendar#436] [Feature Requests]
I'm using obsidian as a Planner (with dataview and so on) and I already have a set of fields that I use for that. So I don't know if others have the same problem, but if you could make a setting for customizable the field names would be awesome.

**Comment by davish** [comment]
Can you post an example of the frontmatter for one of your documents?

**Comment by AtlcatlQ** [comment]
> Can you post an example of the frontmatter for one of your documents?

![imagen](https://user-images.githubusercontent.com/87624226/159348194-73c387da-012b-449e-a0b4-4b0fcb5fb26c.png)

This are the set: `programado` (my version of `startTime` and `date`), `finaliza` (I use this instead of `endTime`, but it should have a full date + time, 'cause Dataview), `limite` (is for due dates), `iniciado` (the date+time when i really started to do it), `completado` (date+time when i completed the task), I also use `cancelado` and `suspendido` for cancelled and suspended tasks.

this is complement of #70 (but they still are diferent features, I know)
